### PR TITLE
Destroy improvements

### PR DESCRIPTION
**Pull request by 6za**
- Makes Telemetry not panic
- Add some UI to Destroy action
- Add Dry-run mode to Destroy

Signed-off-by: 6za <53096417+6za@users.noreply.github.com>

### CODE REVIEW

**Review by pagottoo — DISMISSED** [review]
lgtm

File: cmd/destroy.go [hunk 1]
```diff
@@ -39,6 +41,17 @@ if the registry has already been deleted.`,
 		if err != nil {
 			log.Panic(err)
 		}
+		dryRun, err := cmd.Flags().GetBool("dry-run")
+		if err != nil {
+			log.Panic(err)
```
Comment (converge) [inline cmd/destroy.go:46]: it will avoid stack trace / and will return the error message to the user
```suggestion
			log.Println(err)
			return
```

**Review by converge — DISMISSED** [review]

File: cmd/root.go [hunk 2]
```diff
@@ -15,7 +14,9 @@ var rootCmd = &cobra.Command{
 	open source application delivery platform in under an hour. 
 	checkout the docs at docs.kubefirst.com.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		log.Println(viper.Get("name"))
+		//log.Println(viper.Get("name"))
```
Comment (converge) [inline cmd/root.go:17]:
```suggestion
```

**Review by converge — APPROVED** [review]

File: cmd/destroy.go (hunk 1 above)
Comment (6za) [reply cmd/destroy.go:48]: I am trying to following the existing pattern, not refactor anything at the moment.

File: cmd/root.go (hunk 2 above)
Comment (6za) [reply cmd/root.go:17]: Is this a blocker?

**Review by pagottoo — APPROVED** [review]
lgtm

**Pull request merged**